esp/bindings/http/platform/httpprot.cpp

```diff
@@ -41,6 +41,31 @@
 IThreadPool* http_thread_pool;
 CHttpThreadPoolFactory* http_pool_factory;
 
+struct PooledThreadInfo
+{
+    ISocket& socket;
+    CEspApplicationPort& apport;
+    int maxRequestEntityLength = 0;
+    bool isSSL = false;
+    ISecureSocketContext* ssctx = nullptr;
+    IPersistentHandler* persistentHandler = nullptr;
+    bool shouldClose = false;
+
+    PooledThreadInfo(ISocket& _socket, CEspApplicationPort& _apport) : socket(_socket), apport(_apport) {}
+    ~PooledThreadInfo()
+    {
+#if __cplusplus >= 201703L
```

Comment:
We are always compiling with c++17, so I don't think the #if is necessary.  Is it?

Reply:
Probably not. I found the inverse condition in hqlhtcpp.cpp tried to stay consistent. Will remove.